test/single_file/pre_dec_assign.cc

```diff
@@ -0,0 +1,7 @@
+int main() {
+  int x = 5;
+  --x = 2;
+  volatile int y = 8;
+  --y = 6;
+  return x;
+}
```

Comment:
Same (consider configuring your editor to do this automatically)

Reply:
Done.